stream_file_builder.go

```diff
@@ -54,6 +55,16 @@ const (
 
 var BuiltStreamFileBuilderError = errors.New("StreamFileBuilder has already been built, functions may no longer be used")
 
+var cellTypeToNumFmtMap = map[CellType]string{
+	CellTypeString: builtInNumFmt[builtInNumFmtIndex_STRING],
+	CellTypeNumeric: builtInNumFmt[builtInNumFmtIndex_INT],
+	CellTypeBool: builtInNumFmt[builtInNumFmtIndex_GENERAL],
+	CellTypeInline: builtInNumFmt[builtInNumFmtIndex_STRING],
+	CellTypeError: builtInNumFmt[builtInNumFmtIndex_GENERAL],
+	CellTypeDate: builtInNumFmt[builtInNumFmtIndex_GENERAL],
+	CellTypeStringFormula: builtInNumFmt[builtInNumFmtIndex_STRING],
```

Comment:
Can you document that this is the *default* number format for these types? Previously this library almost entirely conflated the notions of cell type and number format. I've done a lot to distinguish the two concepts in the library, but sometimes it still isn't clear from the code that they are completely different. These are just the default number formats that Excel uses when a cell becomes of a certain cell type.